src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -45,26 +47,165 @@ object ConductrPlugin extends AutoPlugin {
-    super.projectSettings ++ List(
+    List(
       // Here we try to detect what binary universe we exist inside, so we can
       // accurately grab artifact revisions.
       isSbtBuild := Keys.sbtPlugin.?.value.getOrElse(false) && (Keys.baseDirectory in ThisProject).value.getName == "project",
 
+      discoveredDist <<= (dist in Bundle).storeAs(discoveredDist).triggeredBy(dist in Bundle),
+      discoveredConfigDist <<= (dist in BundleConfiguration).storeAs(discoveredConfigDist).triggeredBy(dist in BundleConfiguration)
+    )
+
+  override def buildSettings: Seq[Setting[_]] =
+    List(
+      Keys.aggregate in conduct := false,
+      Keys.aggregate in generateInstallationScript := false,
+      Keys.aggregate in install := false,
+      Keys.aggregate in sandbox := false,
+
       sandbox := sandboxTask.value.evaluated,
       sandboxRunTaskInternal := sandboxRunTask(ScopeFilter(inAnyProject, inAnyConfiguration)).value,
       conduct := conductTask.value.evaluated,
 
-      discoveredDist <<= (dist in Bundle).storeAs(discoveredDist).triggeredBy(dist in Bundle),
-      discoveredConfigDist <<= (dist in BundleConfiguration).storeAs(discoveredConfigDist).triggeredBy(dist in BundleConfiguration)
+      installationData := installationDataTask.value,
+      generateInstallationScript := generateInstallationScriptTask().value,
+      install := installTask().value
     )
 
-  private final val LatestConductrVersion = "1.1.3"
+  private final val LatestConductrVersion = "1.1.5"
   private final val LatestConductrDocVersion = LatestConductrVersion.dropRight(1) :+ "x" // 1.0.0 to 1.0.x
 
   private final val TypesafePropertiesName = "typesafe.properties"
   private final val SandboxRunArgsAttrKey = AttributeKey[SandboxRunArgs]("conductr-sandbox-run-args")
   private final val sandboxRunTaskInternal = taskKey[Unit]("Internal Helper to call sandbox run task.")
 
+  private def installationDataTask: Def.Initialize[Task[Seq[InstallationData]]] = Def.taskDyn {
+    val rootProjectStructure = Project.extract(state.value).structure
+    val allProjects = rootProjectStructure.allProjects
+    val bundleInstallationScriptsTasks =
+      allProjects.map { project =>
+        val projectRef = ProjectRef(rootProjectStructure.root, project.id)
+
+        val bundleConfiguration = project.configurations.find(c => c.name == Bundle.name)
+        val bundleConfigurationConfiguration = project.configurations.find(c => c.name == BundleConfiguration.name)
+
+        (bundleConfiguration, bundleConfigurationConfiguration) match {
+          case (Some(b), Some(bc)) =>
+            Def.taskDyn {
+              val bundleName = (normalizedName in b in projectRef).value
+              val bundleFile = (dist in b in projectRef).value.toPath
+              val bundleConfigContents = (BundleKeys.bundleConf in bc in projectRef).value
+              val srcDir = (sourceDirectory in bc in projectRef).value / (BundleKeys.configurationName in bc in projectRef).value
```

Comment:
Can we rename `srcDir` to `configurationDir` or `bundleConfigSrcDir`? It is the directory of the bundle configuration.